docs/make.jl

```diff
@@ -1,18 +1,25 @@
 Base.HOME_PROJECT[] = abspath(Base.HOME_PROJECT[]) # JuliaLang/julia/pull/28625
 
-using ClimateMachine, Documenter, Literate
-
 # https://github.com/jheinen/GR.jl/issues/278#issuecomment-587090846
 ENV["GKSwstype"] = "100"
+# some of the tutorials cannot be run on the GPU
+ENV["CLIMATEMACHINE_SETTINGS_DISABLE_GPU"] = true
```

Comment:
Is this necessary given that we’ve set this in the Documenter yaml? I think the only tutorial that isn’t GPU friendly is the StateCheck one.

Reply:
this is just convenience (for myself personally) because generating the docs locally doesn't work with the default behavior of deferring to use the GPU by default.

with multiple processes, the ENV distributed to workers is only the process ENV defined at invocation (shell env) not the ENV of the master process during runtime.